tt_metal/llrt/rtoptions.cpp

```diff
@@ -83,6 +83,15 @@ RunTimeOptions::RunTimeOptions() {
 
     const char *validate_kernel_binaries = std::getenv("TT_METAL_VALIDATE_PROGRAM_BINARIES");
     set_validate_kernel_binaries(validate_kernel_binaries != nullptr && validate_kernel_binaries[0] == '1');
+
+    const char *num_cqs = getenv("TT_METAL_NUM_HW_CQS");
+    if (num_cqs != nullptr) {
+        try {
+            set_num_hw_cqs(std::stoi(num_cqs));
+        } catch (const std::invalid_argument& ia) {
+            TT_THROW("Invalid TT_METAL_NUM_HW_CQS: {}", num_cqs);
+        }
+    }
```

Comment:
Do we need docs for this? Seems confusing that there is this env var, but it controls something different from how we set num_hw_cqs during device init. Also not entirely sure what exactly the purpose of this is instead of having it as some sort of arg for where it gets used

Reply:
I didn't actually add this env var, just pulled all of the uses of it into RTOptions. Is this different from how the user sets the number of CQs?

Reply:
Looks like this env var was only ever used in test fixtures

Reply:
Ah okay, this might have just been used in the c++ unit tests before I think. Not sure if it's some relic of how multi cq was originally brought up or not but I don't think we really need (or should) this env var anymore. @aliuTT @tt-asaigal this should be okay to remove/refactored into a test arg instead of env var right?

Users pass in num_hw_cqs as an arg to the CreateDevice(s) api, so devices can be created, closed, and recreated with different number of cqs within one process.

Reply:
Sounds right to me.

Reply:
Does gtest support passing in test arguments from cmd line? Not seeing much on it

Reply:
Seems like we'd have to switch all of our gtests to use `main` instead of `gtest_main`, not sure if we want to do that. For now I've renamed the env var to show that it's gtest specific.
https://groups.google.com/g/googletestframework/c/AmCOYhLHSJw
https://google.github.io/googletest/primer.html#writing-the-main-function